### ISSUE TITLE
Use UTC Timezone

### ISSUE THREAD

**Issue by aneeshusa** [servo/saltfs#303]
This should be applied to all machines, so this should go into the `common/init.sls` file.

We can use the `timezone.system` state to configure a UTC timezone ([docs](https://docs.saltstack.com/en/2015.5/ref/states/all/salt.states.timezone.html)).

I think this would be a great E-Easy issue.

(A follow-up would be ensuring that NTP is on and configured on all machines, but there's no Salt state for this yet so that wouldn't be E-Easy.)

**Comment by RowanEB** [comment]
I'll start working on this.

**Comment by aneeshusa** [comment]
It's all yours! Let me know if you have any questions.

**Comment by RowanEB** [comment]
I think I've correctly added this, should I go about working on adding NTP support, or will that be done from a separate issue?

**Comment by aneeshusa** [comment]
Let's save NTP for another issue/PR; that one will be more involved because we will need to write custom Salt code for that. (It's also blocked on #350, which will enable us to load custom modules and states).

**Comment by aneeshusa** [comment]
@charlesvdv is interested in taking this up now: https://github.com/servo/saltfs/pull/439#issuecomment-269638754

**Comment by jdm** [comment]
Looks like this was fixed by #569.
